test/view_component/view_component_test.rb

```diff
@@ -40,6 +40,13 @@ def test_render_without_template
     assert_selector("input[type='text'][name='name']")
   end
 
+  def test_render_without_template_variant
+    render_inline(InlineComponent.new variants: [:email])
```

Comment:
```suggestion
    render_inline(InlineComponent.new, variants: [:email])
```

It looks like this is passing `variants: [:email]` to the component, not the `render_inline` call